test/box/gh-4703-on_shutdown-bug.result

```diff
@@ -20,9 +20,10 @@ test_run:cmd("setopt delimiter ';'");
  | ---
  | - true
  | ...
+-- Extra \n is required on openSUSE.
```

Comment:
How exactly it relates to openSUSE? Specific readline version? It would be good to reveal some details, if possible.

Reply:
I didn't manage to find out what exactly caused it, but quite possible it is readline6 that is standard on openSUSE.